deployment/centos-package-x64/docker-build.sh

```diff
@@ -8,72 +8,15 @@ set -o xtrace
 # Move to source directory
 pushd ${SOURCE_DIR}
 
-VERSION="$( grep '^Version:' ${SOURCE_DIR}/SOURCES/pkg-src/jellyfin.spec | awk '{ print $NF }' )"
+# Prepare the source
+source "$HOME/.nvm/nvm.sh"
+nvm use v8
+make -f .copr/Makefile srpm outdir=/root/rpmbuild/SRPMS
 
-# Clone down and build Web frontend
-web_build_dir="$( mktemp -d )"
-web_target="${SOURCE_DIR}/MediaBrowser.WebDashboard/jellyfin-web"
-git clone https://github.com/jellyfin/jellyfin-web.git ${web_build_dir}/
-pushd ${web_build_dir}
-if [[ -n ${web_branch} ]]; then
-    checkout -b origin/${web_branch}
-fi
-yarn install
-mkdir -p ${web_target}
-mv dist/* ${web_target}/
-popd
-rm -rf ${web_build_dir}
+# Remove dep for nodejs/yarn since our build env won't have these (NVM instead)
+sed -i '/BuildRequires:  nodejs >= 8 yarn/d' SPECS/jellyfin.spec
```

Comment:
This is really nasty.  A build process should not modify the specfile to suit a platform.  The spec should build untouched on all platforms it's expected to build on.  If there are parts of the spec that are conditional on platform, that needs to be written with conditional statements in the spec.